rpm/pango.spec

```diff
@@ -1,11 +1,10 @@
 Name:       pango
 Summary:    System for layout and rendering of internationalized text
-Version:    1.42.3
+Version:    1.48.10
 Release:    1
-Group:      System/GUI/GNOME
 License:    LGPLv2+
 URL:        http://www.pango.org
-Source0:    http://download.gnome.org/sources/pango/1.42/pango-%{version}.tar.xz
+Source0:    http://download.gnome.org/sources/pango/1.48/pango-%{version}.tar.xz
```

Comment:
This probably could be just %{name}-%{version}.tar.xz which is the way we often have the source in spec files.

Reply:
Adjusted though not sure if it matters much. Pango was even one character shorter :)

Reply:
I think what @mlehtima meant was that you don't need to have that url there. Just `Source0: %{name}-%{version}.tar.xz` would be fine.

Reply:
Yes, I meant that it should have Source0: %{name}-%{version}.tar.xz

Reply:
Got it finally :) Changed.